src/app/components/AssetDetail/index.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { connect } from 'react-redux'
+import { createStructuredSelector } from 'reselect'
+import { compose, setDisplayName, setPropTypes, withProps } from 'recompose'
+import { Card, CardHeader, Row, Col, CardBody, Media, Button } from 'reactstrap'
+import PropTypes from 'prop-types'
+import classNames from 'class-names'
+
+import { getAsset } from 'Selectors/asset'
+import { getHoldingsByAsset } from 'Selectors/wallet'
+import { isPriceChartLoading } from 'Selectors/priceChart'
+
+import Layout from 'Components/Layout'
+import PriceChart from 'Components/PriceChart'
+import CoinIcon from 'Components/CoinIcon'
+import Units from 'Components/Units'
+import ChangePercent from 'Components/ChangePercent'
+import ArrowIcon from 'Components/ArrowIcon'
+
+const getQuery = ({ match }) => match.params.symbol
+
+const marketData = [
+  {
+    title: 'Market Cap',
+    key: 'marketCap',
+  },
+  {
+    title: '24hr Volume',
+    key: 'volume24',
+  },
+  {
+    title: 'Supply',
+    key: 'availableSupply',
+  }
+]
+
+let TO_ASSET = 'BTC'
+let FROM_ASSET = 'ETH'
+
+const AssetDetail = ({ symbol, asset, assetHoldings, isPriceChartLoading }) => {
+  const { name, price, change24, deposit, receive } = asset
+  if (symbol === 'BTC') { TO_ASSET = 'ETH' }
+  if (symbol === 'ETH') { FROM_ASSET = 'BTC' }
+  return (
+    <Layout className='pt-3 p-0 p-sm-3'>
+      <Card>
+      <CardHeader className='grid-group'>
+        <Row className='gutter-3 p-sm-0 p-3 d-flex'>
+          <Col className='d-flex align-items-center pl-sm-4 pl-0 py-2 col-auto' size='sm'>
+            <Media>
+              <Media left>
+                <CoinIcon 
+                  className='mr-2 mt-1' 
+                  symbol={symbol} 
+                  style={{ width: '40px', height: '40px', position: 'relative', top: '2px' }} 
+                  inline
+                /> 
+              </Media>
+              <Media body>
+                <Media className='m-0 mt-1 font-weight-bold' heading>
+                  {name}
+                </Media>
+                <small style={{ position: 'relative', top: '-5px' }} className='text-muted'>[{symbol}]</small>
+              </Media>
+            </Media>
+          </Col>
+          <Col className='col-sm-8 col-md-8 col-lg-2 col-5'>
+            <div className='pl-4 py-2'>
+              <div className='mb-0'>
+                <Units 
+                  className='mt-1 d-inline-block font-weight-bold'
+                  value={price} 
+                  symbol={'$'} 
+                  precision={6} 
+                  prefixSymbol
+                />
+              </div>
+              <small style={{ position: 'relative', top: '-5px' }}><ChangePercent>{change24}</ChangePercent></small>
```

Comment:
Do we need a label so users know it's past 24hr change? Or maybe in an `Expandable`?